client/tests/Spotlight.test.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Spotlight from '../src/Spotlight';
 import { shallow } from 'enzyme';
+import sinon from "sinon";
 
 describe('Unit Test for <Spotlight /> Component', () => {
   it('should render a div with class name "spotlight"', () => {
```

Comment:
You should consider changing the description for the test. It seems to be testing a prop.

Reply:
@tremarkley - I'm unsure about the test you are referring to here. Is it the test on line 7, 'should render a div with class name "spotlight"'? I thought in using expect and hasClass I would be testing classname and not prop, but let me know if I'm misunderstanding something here, thanks!